### DISCUSSION TITLE
ScrollView 支持 touchmove 事件冒泡

### DISCUSSION THREAD

**Discussion by bluescurry** [NervJS/taro#15065] [Q & A] [answered]
### 这个特性解决了什么问题？
我使用 ScrollView 的 touchMove 冒泡实现划屏功能，在 3.6.17 版本 ScrollView 对 touchMove 事件增加阻止冒泡，导致我的功能无法使用了。
https://github.com/NervJS/taro/blob/main/packages/taro-components/src/components/scroll-view/scroll-view.tsx#L125

### 这个 API 长什么样？
我希望能增加一个配置，允许设置 touchMove 事件不阻止冒泡，默认可以阻止冒泡。

**Comment by ZakaryCode** [comment, marked as answer]
这对于 touchMove 事件冒泡应该并不是必须？在内部套一层 View 获取 touchMove 事件，或者 ScrollView 监听事件抛出都能达成需求

**Comment by bluescurry** [comment]
> 这对于 touchMove 事件冒泡应该并不是必须？在内部套一层 View 获取 touchMove 事件，或者 ScrollView 监听事件抛出都能达成需求

@ZakaryCode 我举个例子：
```html
 <View
  catchMove
  onTouchMove={() => {}}
>
  <ScrollView
     onTouchMove={() => {}}
  ></ScrollView>
</View>
```
在 3.6.16 版本，在滚动 scrollview 时，外层的 view 可以触发 onTouchMove 事件，在3.6.17 因为 stopPropagation 阻止了冒泡，无法触发外层 view 的 onTouchMove 事件

**Comment by ZakaryCode** [comment]
并没有错误理解你的需求，ScrollView 抛出事件隐患还是有比较多，新增参数支持必要性不足，还是建议按照上述的方案调整即可。

**Comment by Rambos** [comment]
这种如果想做交互的话基本没法搞(比如利用`swiper`来做左右滑动切换，同时`swiper-item`内容支持滚动)，只能用`View>overflow-y:auto`了。
